src/main/java/frc/robot/subsystems/vision/coordinate/CoordinateDistance.java

```diff
@@ -0,0 +1,34 @@
+package frc.robot.subsystems.vision.coordinate;
+
+import frc.robot.subsystems.vision.provider.Vision;
+
+public class CoordinateDistance implements Distance {
+
+    private final Vision.Coordinates coordinates;
+    private final double targetHeight;
+    private final double cameraHeight;
+    private final double cameraAngle;
+
+    public CoordinateDistance(Vision.Coordinates coordinates, double targetHeight, double cameraHeight, double cameraAngle) {
+        this.coordinates = coordinates;
+        this.targetHeight = targetHeight;
+        this.cameraHeight = cameraHeight;
+        this.cameraAngle = cameraAngle;
+    }
+
+    //default constructor. up to user to decide whether they want to use some configuration library to provide targetHeight.. etc or just use constants
```

Comment:
If you type `/**` into vscode or IntelliJ it should automatically generate javadoc comments above methods.

Reply:
Though as I read other parts of your PR, you obviously know this. Is there a reason these comments aren't using javadoc comment strings?

Reply:
These comments aren't for "api" users to read - they are reminders to me and the reviewer. They will be replaced with proper documentation as the subsystem matures